tests/integration/test_dbapi_integration.py

```diff
@@ -211,7 +211,7 @@ def test_experimental_python_types_with_connection_and_cursor(
     """)
     rows = cur.fetchall()
 
-    if expected:
+    if not expected:
```

Comment:
Maybe it makes sense to rename `expected` into something more explicit, eg `legacy_primitive_types_expected`

Reply:
renamed to `expected_legacy_primitive_types`